google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/TestEnvRule.java

```diff
@@ -122,9 +125,14 @@ protected void before(Description description) throws Throwable {
   }
 
   private void configureLogging(Description description) throws IOException {
-    if (Strings.isNullOrEmpty(BIGTABLE_GRPC_LOG_DIR)) {
+    if (!BIGTABLE_ENABLE_VERBOSE_GRPC_LOGS) {
       return;
     }
+    Preconditions.checkState(
```

Comment:
should we be setting this in the kokoro configs?

Reply:
Yes, it will be set in the kokoro directpath tests via the `enable-verbose-grpc-logs` profile